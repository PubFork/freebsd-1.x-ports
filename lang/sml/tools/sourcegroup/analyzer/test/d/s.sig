signature SCOPES = sig
 structure MD :MODULE_DECLS

 type moduleInfo
 type scope

 val init       :unit -> unit
 val topScope   :unit -> scope
 val bindScope  :scope -> scope
 val localScope :scope -> scope
 val plainScope :scope -> scope
 val popScope   :scope -> scope
 val enterRef   :MN.moduleName * scope -> scope
 val enterDef   :MN.moduleName * scope -> scope
 val lookup     :scope -> MN.moduleName -> moduleInfo option

 val printAll   :outstream -> unit
end
