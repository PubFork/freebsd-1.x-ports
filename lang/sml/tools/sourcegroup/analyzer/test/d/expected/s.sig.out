source sml "d/s.sig"
  import structure MN 
  import signature MODULE_DECLS 
  export signature SCOPES ;

